sphinx/writers/latex.py

```diff
@@ -275,6 +275,22 @@ def escape_abbr(text):
     return re.sub('\.(?=\s|$)', '.\\@', text)
 
 
+def rstdim_to_latexdim(width_str):
+    """Convert `width_str` with rst length to LaTeX length."""
+    match = re.match('^([0-9.]+)\s*(\S*)$', width_str)
+    if not match:
+        raise ValueError
+    res = width_str
+    amount, unit = match.groups()[:2]
+    if unit in ('', "px"):
+        res = "%s\\sphinxpxdimen" % amount
+    elif unit == 'pt':
+        res = '%sbp' % amount  # convert to 'bp'
```

Comment:
memo: the reason for doing this is that Sphinx is multi-target and `pt` everywhere means `1/72in` except in TeX, where it means `1/72.27in`. The DocUtils latex writer does this conversion too. http://docutils.sourceforge.net/docs/user/latex.html#length-units